libs/SalesforceSDKCommon/SalesforceSDKCommon/Classes/Logger/SFLogger.m

```diff
@@ -67,82 +105,132 @@ -(void)setLogLevel:(SFLogLevel) logLevel {
 }
 - (void)log:(nonnull Class)cls message:(nonnull NSString *)message {
     [self.logger log:cls level:SFLogLevelDefault message:message];
+    
+    [self.logReceiver receiveWithLevel:SFLogLevelDefault cls:cls component:self.logger.componentName message:message];
```

Comment:
Here's the parallel logging for the new log receiver.  There's quite a few points for this.

Reply:
Could the parallel calls be consolidated to a few helper methods (like one each for message, format, etc) that the top levels could call into?

Reply:
I just pushed a commit to that effect 🌈